css/mistica-common.css

```diff
@@ -44,6 +44,8 @@
     --vcolor-checkbox-tick: var(--mistica-color-controlKnobInverse);
     --vcolor-checkbox-background: transparent;
 
+    --vcolor-inputBorder: var(--mistica-color-backgroundContainer);
```

Comment:
Seems that the PR https://github.com/Telefonica/mistica-web/pull/1337 missed the border when the counter is inverse and in dark mode, since the border is visible when is a default context and in dark mode.

Both should look the same, the correct one is the current default context with the visible border.

Reply:
should be fixed